hrjobcontract/js/src/job-contract/controllers/form/form-hour.controller.js

```diff
@@ -24,7 +24,7 @@ define([
     $scope.hrsAmountDefined = false;
 
     (function init () {
-      entityHour.location_standard_hours = entityHour.location_standard_hours || '1';
+      entityHour.location_standard_hours = entityHour.location_standard_hours || $scope.utils.hoursLocation[0].id;
```

Comment:
what will happen if the hours location array is empty? Lets use lodash's get method which makes accessing undefined object/array properties safe:

```js
entityHour.location_standard_hours = entityHour.location_standard_hours || _.get(utilsHoursLocation, '[0].id');
```

`utilsHoursLocation` is defined above.

Reply:
@reneolivo sorry, what would that return if `utilsHoursLocation` is empty?